app/src/main/java/crux/bphc/cms/models/Course.java

```diff
@@ -114,7 +114,7 @@ public String getShortname() {
 
     public String[] getCourseName(){
         String courseName = getShortname();
-        String regex =  "([\\w\\d \\-'&,]+ \\w\\d\\d\\d) ([\\w\\d \\-'&,]+) ([LTP]\\d*)";  // Specifies the string pattern which is to be searched
+        String regex =  "([\\w\\d \\-/'&,]+ \\w\\d\\d\\d) ([\\w\\d \\-/():+\"'&.,?\\[\\]]+) ([LTP]\\d*)";  // Specifies the string pattern which is to be searched
```

Comment:
Any specific reason as to why `/():+\"'&.,?\\[\\]` was also included? Are characters like paranthesis and `?` seen in course names?

Reply:
Parenthesis have been used in certain course names like "CE F243 SOIL MECHANICS T1 (SAT)".
Adding other characters was probably unnecessary. I'll change that.

Reply:
Remove the square brackets. I don't think those will be used.

Reply:
Everything can be left behind.